orders.asp

```diff
@@ -229,6 +229,7 @@ Class YQuery
 					& " ROW_NUMBER() OVER(ORDER BY OrderDate) as Row" _
 					& " FROM Orders" _				
 					& " WHERE OrderDate between '{StartDate}' and '{EndDate}'" _
+					& " OR ShipDate between '{StartDate}' and '{EndDate}'" _

```

Comment:
We should get a param in the request which does the distinction between ShipDate and OrderDate